utilities/BuildUtilities.groovy

```diff
@@ -139,32 +149,57 @@ def copySourceFiles(String buildFile, String srcPDS, String dependencyPDS, Depen
 		}
 		if (props.verbose) println "*** Physical dependencies for $buildFile:"
 
+		// Load property mapping containing the map of targetPDS and dependencyfile
+		PropertyMappings dependenciesDatasetMapping = new PropertyMappings(dependencyDatasetMapping)
+		
 		physicalDependencies.each { physicalDependency ->
 			if (props.verbose) println physicalDependency
+
 			if (physicalDependency.isResolved()) {
+
+				// obtain target dataset based on Mappings
+				// Order :
+				//    1. langprefix_dependenciesAlternativeLibraryNameMapping based on the library setting recognized by DBB (COBOL and PLI)
+				//    2. langprefix_dependenciesDatasetMapping as a manual overwrite to determine an alternative library used in the default dd concatentation 
+				String dependencyPDS 
+				if (!physicalDependency.getLibrary().equals("SYSLIB") && dependenciesAlternativeLibraryNameMapping) {
+					dependencyPDS = props.getProperty(evaluate(dependenciesAlternativeLibraryNameMapping).get(physicalDependency.getLibrary()))
+				}
+				if (dependencyPDS == null && dependenciesDatasetMapping){
+					dependencyPDS = props.getProperty(dependenciesDatasetMapping.getValue(physicalDependency.getFile()))
+				}
+				physicalDependency.getLibrary()
```

Comment:
What is the goal if this line? This is just a call to a method with no assignment.

Reply:
Removed .